src/components/users.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { Users } from '../mocks/users'
+
+class UsersList extends Component {
+	constructor(){
+		super();
+		this.state = {
+			data: Users
```

Comment:
what if you need another field in your state?
what name should it be? data2?
better rename data => users